### ISSUE TITLE
notepadqq crashes on new install with missing config files

### ISSUE THREAD

**Issue by severach** [notepadqq/notepadqq#440]
```
[x] bug report
```

**Current behavior**
As of 2017-04-20 21:04 7a4e887 notepadqq crashes on a new install where `~/.config/Notepadqq` is not fully populated.

**Minimal reproduction of the problem with instructions**
````
% rm -rf ~/.config/Notepadqq
% strace notepadqq
...
stat("/etc/xdg/Notepadqq.ini", 0x7ffc4c3139a0) = -1 ENOENT (No such file or directory)
open("/home/chris/.config/Notepadqq/session.xml", O_RDONLY|O_CLOEXEC) = -1 ENOENT (No such file or directory)
--- SIGSEGV {si_signo=SIGSEGV, si_code=SEGV_MAPERR, si_addr=0x51} ---
+++ killed by SIGSEGV (core dumped) +++
[1]    6304 segmentation fault (core dumped)  strace notepadqq
````

It will not crash if I install an older version, perform some editing, then install the newer version.

**Please tell us about your environment:**
````
% pacman -Q linux notepadqq-git qt5-base                                       :(
linux 4.10.13-1
notepadqq-git 1.0.1.r42.ge420041-1
qt5-base 5.8.0-11
````

**Comment by Teklad** [comment]
It actually has nothing to do with files missing.  We stupidly forgot to check if the editor is valid here. Lol.

**Comment by danieleds** [comment]
Thank you! Fixed in 0bfc1bd59e9bb891be3e3249525961047ed304c1
